README.md

```diff
@@ -27,13 +27,13 @@ two reconcilers:
 1. Build and push your image to the location specified by `IMG`:
 	
 ```sh
-make docker-build docker-push IMG=<some-registry>/controller-runtime-example:tag
+make docker-build docker-push REGISTRY=<some-registry> IMG=controller-runtime-example:tag
```

Comment:
This is taken from KB: https://book.kubebuilder.io/quick-start.html?highlight=make,docker#run-it-on-the-cluster. <registry> is usually a part of image specification.